pending-documentation.md

```diff
@@ -457,6 +457,94 @@ A single configuration cell containing an integer may have the "exit"
 attribute. This integer will then be used as the return value on the console
 when executing the program.
 
+For example:
+
+```k
+configuration <k> $PGM:Pgm </k>
+              <status-code exit=""> 1 </status-code>
+```
+
+declares that the cell `status-code` should be used as the exit-code for
+invocations of `krun`. Additionally, we state that the default exit-code is `1`
+(an error state). One use of this is for writing testing harnesses which assume
+that the test fails until proven otherwise and only set the `<status-code>` cell
+to `0` if the test succeeds.
+
+### Collection Cells: `multiplicity` and `type` attributes
+
+Sometimes a semantics needs to allow multiple copies of the same cell, for
+example if you are making a concurrent multi-threading programming language.
+For this purpose, K supports the `multiplicity` and `type` attributes on cells
+declared in the configuration.
+
+`multiplicity` can take on values `*` and `?`. Declaring `multiplicity="*"`
+indicates that the cell may appear any number of times in a runtime
+configuration. Setting `multiplicity="?"` indicates that the cell may only
+appear exactly 0 or 1 times in a runtime configuration.
```

Comment:
Maybe we should mention the effect that `*` and `?` have on building the initial configuration.

Reply:
Added, please check it.